frontend/src/main/components/Users/UsersTable.js

```diff
@@ -1,33 +1,102 @@
 import React from "react";
 import OurTable from "main/components/OurTable"
-
-const columns = [
-    {
-        Header: 'id',
-        accessor: 'id', // accessor is the "key" in the data
-    },
-    {
-        Header: 'First Name',
-        accessor: 'givenName',
-    },
-    {
-        Header: 'Last Name',
-        accessor: 'familyName',
-    },
-    {
-        Header: 'Email',
-        accessor: 'email',
-    },
-    {
-        Header: 'Admin',
-        id: 'admin',
-        accessor: (row, _rowIndex) => String(row.admin) // hack needed for boolean values to show up
-    },
-];
+import { Button } from "react-bootstrap";
+import { useBackendMutation } from "main/utils/useBackend";
+import { cellToAxiosParamsDelete, cellToAxiosParamsUnhide, onDeleteSuccess } from "main/utils/usersUtils"
 
 export default function UsersTable({ users }) {
+
+    // Stryker disable all : hard to test for query caching
+    const deleteMutation = useBackendMutation(
+        cellToAxiosParamsDelete,
```

Comment:
all these references to `delete` should be `hide` yes?

Naming is important!   Studies have found that code with misleading names ends up with higher bug rates.